src/components/InstantSearch.vue

```diff
@@ -103,6 +104,9 @@ export default {
       );
     },
   },
+  created() {
+    addAlgoliaAgent(this.searchClient);
```

Comment:
Since the logic is short & simple, I would rather go for inline it. We are not gonna re-use this logic anywhere in the code.